modules/ebdrBidAdapter.js

```diff
@@ -0,0 +1,119 @@
+      requestId = bid.bidderRequestId;
+      ebdrImps.push({
+        id: bid.bidId,
+        [_mediaTypes]: {
+          w: whArr[0],
+          h: whArr[1]
+        },
+        bidfloor: bidFloor
+      })
+      ebdrReq[bid.bidId] = {mediaTypes: _mediaTypes,
+        w: whArr[0],
+        h: whArr[1]
+      };
+      ebdrParams['latitude'] = utils.getBidIdParameter('latitude', bid.params);
+      ebdrParams['longitude'] = utils.getBidIdParameter('longitude', bid.params);
+      ebdrParams['ifa'] = (utils.getBidIdParameter('IDFA', bid.params).length > utils.getBidIdParameter('ADID', bid.params).length) ? utils.getBidIdParameter('IDFA', bid.params) : utils.getBidIdParameter('ADID', bid.params);
+    });
+    let ebdrBidReq = {
+      id: requestId,
+      imp: ebdrImps,
+      site: {
+        domain: domain,
+        page: page
+      },
+      device: {
+        geo: {
+          lat: ebdrParams.latitude,
+          log: ebdrParams.longitude
+        },
+        ifa: ebdrParams.ifa
+      }
+    };
+    return {
+      method: 'GET',
+      url: '//' + rtbServerDomain + '/hb?' + '&zoneid=' + zoneid + '&br=' + encodeURIComponent(JSON.stringify(ebdrBidReq)),
+      bids: ebdrReq
+    };
+  },
+  interpretResponse: function(serverResponse, bidRequest) {
+    utils.logInfo('Log serverResponse', serverResponse);
+    utils.logInfo('Log bidRequest', bidRequest);
+    let ebdrResponseImps = [];
+    const ebdrResponseObj = serverResponse.body;
+    if (!ebdrResponseObj || !ebdrResponseObj.seatbid || ebdrResponseObj.seatbid.length === 0 || !ebdrResponseObj.seatbid[0].bid || ebdrResponseObj.seatbid[0].bid.length === 0) {
+      return [];
+    }
+    ebdrResponseObj.seatbid[0].bid.forEach(ebdrBid => {
+      let responseCPM;
+      responseCPM = parseFloat(ebdrBid.price);
+      let adm;
+      let type;
+      if (bidRequest.bids[ebdrBid.id].mediaTypes == BANNER) {
+        adm = decodeURIComponent(ebdrBid.adm)
+        type = 'ad';
+      } else {
+        adm = ebdrBid.adm
+        type = 'vastXml'
+      }
+      ebdrResponseImps.push({
+        requestId: ebdrBid.id,
+        [type]: adm,
+        creativeId: ebdrBid.crid,
+        cpm: responseCPM,
+        width: ebdrBid.w,
+        height: ebdrBid.h,
+        currency: 'USD',
+        netRevenue: true,
+        ttl: 3600 });
+    });
+    return ebdrResponseImps;
+  }
+}
+function getWidthAndHeight(bid) {
+  let adW = null;
+  let adH = null;
+
+  if (bid.sizes && bid.sizes.length) {
+    let sizeArrayLength = bid.sizes.length;
+    if (sizeArrayLength === 2 && typeof bid.sizes[0] === 'number' && typeof bid.sizes[1] === 'number') {
+      adW = bid.sizes[0];
+      adH = bid.sizes[1];
+    }
+  }
+  let _mediaTypes = bid.mediaTypes && bid.mediaTypes.video ? VIDEO : BANNER;
+  if (_mediaTypes == BANNER && bid.mediaTypes[_mediaTypes].sizes && bid.mediaTypes[_mediaTypes].sizes[0] && bid.mediaTypes[_mediaTypes].sizes[0].length === 2) {
```

Comment:
This `if` statement doesn't verify if the `bid.mediaTypes` and `bid.mediaTypes[_mediaTypes]` objects exist.  This could lead to `undefined` errors if the page only had `adUnit.sizes` defined (which may be the case with some older prebid.js supported sites).  

The earlier statement for the `_mediaTypes` variable doesn't guarantee the `bid.mediaTypes` object actually exists when it assigns `BANNER`.

Should alter the logic here (likely in the `if` but perhaps in `_mediaTypes` to avoid potential errors.